src/Xamarin.Android.Build.Tasks/Tasks/Aapt2Link.cs

```diff
@@ -148,8 +148,16 @@ string GenerateCommandLineCommands (string ManifestFile, string currentAbi, stri
 			}
 
 			if (CompiledResourceFlatArchive != null) {
-				var flata = Path.Combine (WorkingDirectory, CompiledResourceFlatArchive.ItemSpec);
-				if (File.Exists (flata)) {
+				var flata = CompiledResourceFlatArchive.ItemSpec;//Path.Combine (WorkingDirectory, CompiledResourceFlatArchive.ItemSpec);
```

Comment:
The space issue is tough. What if `$(Configuration)` has a space in it? Like `Configuration=Google Play`. I've seen devs do it before. 🤦‍♂

Reply:
Indeed those tests are failing :) So I'm working on something else :P